streams/src/main/java/org/apache/kafka/streams/state/ReadOnlyWindowStore.java

```diff
@@ -77,4 +77,23 @@
      * @throws NullPointerException If null is used for any key.
      */
     KeyValueIterator<Windowed<K>, V> fetch(K from, K to, long timeFrom, long timeTo);
+    
+    /**
+    * Gets all the key-value pairs in the existing windows.
+    *
+    * @returns an iterator over windowed key-value pairs {@code <Windowed<K>, value>}
+    * @throws InvalidStateStoreException if the store is not initialized
+    */
+    KeyValueIterator<Windowed<K>, V> all();
+    
+    /**
+     * Gets all windows for a specific time range
```

Comment:
Nit: Gets all the key-value pairs that belong to the windows within in the given time range.